examples/v1beta1/pipelineruns/alpha/pipeline-with-when-expressions-using-array-results.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: pipelinerun-with-when-expressions-using-array-results-
+spec:
+  pipelineSpec:
+    params:
+      - name: images
+        type: array
+        description: The list of images
+    tasks:
+      - name: task1
+        taskSpec:
+          results:
+            # a list of images in array-result-1
+            - name: array-results-1
+              type: array
+              description: The array results
+            # a list of images in array-result-2
+            - name: array-results-2
+              type: array
+              description: The array results
```

Comment:
looks like this result - `array-results-2` - is neither produced in this task nor used in subsequent tasks 🤔

Reply:
yeah I just left it to make sure such result declaration does not impact here in any way, but on a second thought, may be not necessary here, deleted it.